tests/integration/api_build_test.py

```diff
@@ -448,8 +448,10 @@ def test_build_invalid_platform(self):
             for _ in stream:
                 pass
 
-        assert excinfo.value.status_code == 400
-        assert 'invalid platform' in excinfo.exconly()
+        # Some API versions incorrectly returns 500 status; assert 400 or 500
+        assert excinfo.value.is_error()
```

Comment:
If I understand correctly this will check for any error, the comment is misleading, either we change the comment or we test for the code to be 400 or 500

Reply:
Well, it tests for 400 or 500; https://github.com/thaJeztah/docker-py/blob/219c52141e3cd15db3348c5420220f640323499f/tests/unit/errors_test.py#L26-L38

Reply:
Whoops; wrong link: https://github.com/docker/docker-py/blob/219c52141e3cd15db3348c5420220f640323499f/docker/errors.py#L66-L77

Reply:
Can change it to `4xx` or `5xx`

Reply:
Isn't this that is called when you check for `is_error()`? 
https://github.com/docker/docker-py/blob/1073b7364846709ad005e7991045d38365d911c9/docker/errors.py#L66-L77

Reply:
@rumpl Yes, sorry, linked to the wrong part initially (see comments above)

changed it to `4xx or 5xx` in the comment